src/items/itemDialog.component.tsx

```diff
@@ -759,26 +756,29 @@ function ItemDialog(props: ItemDialogProps) {
               </FormControl>
             </Grid>
             <Grid item xs={12}>
-              <FormControl size="small" fullWidth>
-                <InputLabel required={true} id="usage-status">
-                  Usage status
-                </InputLabel>
-                <Select
-                  required={true}
-                  labelId="usage-status"
-                  value={itemDetails.usage_status ?? 'new'}
-                  size="small"
-                  onChange={(e) =>
-                    handleItemDetails('usage_status', e.target.value)
-                  }
-                  label="Usage status"
-                >
-                  <MenuItem value={'new'}>New</MenuItem>
-                  <MenuItem value={'inUse'}>In Use</MenuItem>
-                  <MenuItem value={'used'}>Used</MenuItem>
-                  <MenuItem value={'scrapped'}>Scrapped</MenuItem>
-                </Select>
-              </FormControl>
+              {usageStatuses && (
```

Comment:
Is it possible to make this show all the time - on a slow connection it adds to the form after a while which I think is also an accessibility issue, it should be there but just not display any options until loaded.